### PR TITLE
User marker

### PR DESCRIPTION
**Pull request by ElectricZebra**
Added user marker

### CODE REVIEW

File: src/components/MapDisplay.js [hunk 1]
```diff
@@ -43,18 +44,18 @@ class MapDisplay extends React.Component {
...
   		heatmapCoords: [ ...this.state.heatmapCoords, {lat, lng}]
   	})
-    if (this._googleMap !== undefined) {      
+    if (this._googleMap !== undefined) {
       const point = new google.maps.LatLng(lat, lng)
       this._googleMap.heatmap.data.push(point)
     }
   }
-  toggleHeatMap() {    
+  toggleHeatMap() {
```
Comment (skythrilla) [inline src/components/MapDisplay.js]: Why are you altering these lines for no reason?

Comment (ElectricZebra) [reply src/components/MapDisplay.js]: I'm not quite sure. I don't even think I touched those lines. Perhaps it was my linter?

Comment (ElectricZebra) [reply src/components/MapDisplay.js]: Sorry about that I'll see if I can revert

Comment (ElectricZebra) [reply src/components/MapDisplay.js]: I'm pretty sure that was ESLint removing trailing whitespace at the end of lines

Comment (BAlvernaz) [reply src/components/MapDisplay.js]: It looks like all Nick did was remove a a few spaces, Definetly a linter.

**Pull request closed without merging**